docs/typescript/generating-types.mdx

```diff
@@ -18,6 +18,32 @@ payload generate:types
 
 You can run this command whenever you need to regenerate your types, and then you can use these types in your Payload code directly.
 
+### Disable declare statement
+
+By default, `generate:types` will add a `declare` statement to your types file in order to better assist with type inference.
+
+If you would like to disable this, you can add the `declare` property as false under the `typescript` settings.
```

Comment:
If you are using your `payload-types.ts` file in other repos, though, it might be better to disable this `declare` statement, so that you don't get any TS errors in projects that use your Payload types, but do not have Payload installed.